stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/ChatDomainImpl.kt

```diff
@@ -722,7 +728,7 @@ internal class ChatDomainImpl internal constructor(
 
     @VisibleForTesting
     internal suspend fun retryMessages(): List<Message> {
-        return retryMessagesWithoutAttachments() + retryMessagesWithAttachments()
+        return retryMessagesWithSyncedAttachments() + retryMessagesWithAttachments()
```

Comment:
Could we do some further renaming here to clearify what the two categories we're handling are?

```suggestion
        return retryMessagesWithSyncedAttachments() + retryMessagesWithPendingAttachments()
```